{"uuid": "9bfa4251-595d-4b77-afa9-c648ce16628a", "children": ["f53ac02c-02ad-4784-a962-51bc37fab28d"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 90\nCurrent browser version is 92.0.4515.107 with binary path C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\n\n", "trace": "  File \"C:\\Users\\Владимир\\environments\\selenium_env\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Владимир\\environments\\selenium_env\\lib\\site-packages\\_pytest\\fixtures.py\", line 918, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Владимир\\environments\\selenium_env\\lib\\site-packages\\_pytest\\fixtures.py\", line 774, in call_fixture_func\n    res = next(it)\n  File \"C:\\Users\\Владимир\\PycharmProjects\\autotest_staging\\CS\\conftest.py\", line 27, in browser\n    browser = webdriver.Chrome(options=options)\n  File \"C:\\Users\\Владимир\\environments\\selenium_env\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 75, in __init__\n    desired_capabilities=desired_capabilities)\n  File \"C:\\Users\\Владимир\\environments\\selenium_env\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 156, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"C:\\Users\\Владимир\\environments\\selenium_env\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 251, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"C:\\Users\\Владимир\\environments\\selenium_env\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 320, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Владимир\\environments\\selenium_env\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1627560017498, "stop": 1627560022766}], "start": 1627560017498, "stop": 1627560022970}